### PR TITLE
Feature/issue 1067 Add Covid0-19 Information

### PR DESCRIPTION
**Pull request by niko-pg**

### CODE REVIEW

File: src/components/recommended_topics/call_to_action.tsx [hunk 1]
```diff
@@ -1,14 +1,82 @@
...
+                    service delivery methods. Please check service providers'
+                    websites for details.
+                </Trans>
+            </Text>
+            <Text style={[textStyles.paragraphStyleBrown, { marginBottom: 20 }]}>
+                <Trans>
+                    Questions about COVID-19? Get the latest information from
+                    <Text>{' '}</Text>
+                    <Text style={markdownStyles.link} onPress={onLinkPress}>
+                        BC Centre for Disease Control
```
Comment (tomy-pg) [inline src/components/recommended_topics/call_to_action.tsx]: nit-picky detail, but you are missing a period after `BC Centre for Disease Control`

File: src/components/recommended_topics/recommended_topics_component.tsx [hunk 2]
```diff
@@ -17,6 +17,7 @@ import {
 import { RecommendedIconComponent } from './recommended_icon_component';
 import { buildTopicsListItemsWithHeadings } from '../topics/build_topic_list_items_with_headings';
 import { EmptyTopicListComponent } from '../empty_component/empty_topic_list_component';
+import { AnalyticsLinkPressedAction } from '../../stores/analytics';
```
Comment (rasmus-storjohann-PG) [inline src/components/recommended_topics/recommended_topics_component.tsx]: Typically I would squish this commit into the previous one. All commits should build successfully and all tests should pass.

**Comment by rasmus-storjohann-PG** [comment]
This is good to go in if the localization works

**Comment by tomy-pg** [comment]
Closing this and cherry picking these commits to merge this against develop.

**Pull request closed without merging**